ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/ClusterTerminator.java

```diff
@@ -99,8 +100,13 @@ private void deleteTopics(final Collection<String> topicsToBeDeleted) {
               filterNonExistingTopics(topicsToBeDeleted)),
           ExecutorUtil.RetryBehaviour.ALWAYS);
     } catch (final Exception e) {
-      throw new KsqlException(
-          "Exception while deleting topics: " + StringUtils.join(topicsToBeDeleted, ", "));
+      // An exception due to TopicDeletionDisabledException should be ignored when a Cluster
+      // termination is requested. The deleteTopics already logs an INFO message. We can skip
+      // logging it here.
+      if (!(e instanceof TopicDeletionDisabledException)) {
```

Comment:
Again, nasty - clean up by not logging in `deleteTopics`.

Reply:
Done